templates/terraform/examples/url_map_traffic_director_route.tf.erb

```diff
@@ -0,0 +1,85 @@
+resource "google_compute_url_map" "<%= ctx[:primary_resource_id] %>" {
+  provider = "google-beta"
+  name        = "<%= ctx[:vars]['url_map_name'] %>"
+  description = "a description"
+  default_service = "${google_compute_backend_service.home.self_link}"
+
+  host_rule {
+    hosts        = ["mysite.com"]
+    path_matcher = "allpaths"
+  }
+
+  path_matcher {
+    name = "allpaths"
+    default_service = "${google_compute_backend_service.home.self_link}"
+
+    route_rules {
+      priority = 1
+      header_action {
```

Comment:
Can you edit an example of this field so that only some elements are set and not all of them?